### PR TITLE
Bump version to 1.3.1

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Bump version to 1.3.1

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/1805

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-reports/opensearch_dashboards.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "reportsDashboards",
-  "version": "1.3.0.0",
+  "version": "1.3.1.0",
   "opensearchDashboardsVersion": "1.3.0",
```
Comment (joshuali925) [inline dashboards-reports/opensearch_dashboards.json:4]: dashboards core not 1.3.1?

**Review by joshuali925 — APPROVED** [review]
does reports-scheduler (reporting opensearch plugin) also need the bump?

File: dashboards-reports/opensearch_dashboards.json (hunk 1 above)
Comment (gaiksaya) [reply dashboards-reports/opensearch_dashboards.json:4]: No just opensearch component

Comment (joshuali925) [reply dashboards-reports/opensearch_dashboards.json:4]: then this PR should not bump reportsDashboards?

**Comment by gaiksaya** [comment]
Thanks of rpointing that out @joshuali925 . Please see https://github.com/opensearch-project/dashboards-reports/pull/324 for updated PR. Thanks!

**Pull request closed without merging**